docs/deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
+
+# abort on errors
+set -e
+
+# build
+yarn run dist
+
+# navigate into the build output directory
+cd temp/.vuepress/dist
+
+# if you are deploying to a custom domain
```

Comment:
Can you better specify these comments? The "if" seems that this is a configuration and we can set up to a custom domain or not, but I don't believe it is the case.

Reply:
Done in 5b3bbc8